qiita_db/util.py

```diff
@@ -727,6 +729,36 @@ def purge_filepaths(conn_handler=None):
                 remove(fp)
 
 
+def move_filepaths_to_upload_folder(study_id, filepaths, conn_handler=None):
+    r"""Goes over the filepaths list and moves all the filepaths that are not
+    used in any place to the upload folder of the study
+
+    Parameters
+    ----------
+    study_id : int
+        The study id to where the files should be returned to
+    filepaths : list
+        List of filepaths to move to the upload folder
+    conn_handler : SQLConnectionHandler, optional
+        The connection handler object connected to the DB
+    """
+    conn_handler = conn_handler if conn_handler else SQLConnectionHandler()
+    uploads_fp = join(get_mountpoint("uploads")[0][1], str(study_id))
+
+    # We can now go over and remove all the filepaths
+    for fp_id, fp, fp_type in filepaths:
```

Comment:
`fp_type` is not being used anywhere in this loop, for clarity it may make sense to put `_`. Not a blocking comment though.